junit-platform-launcher/src/main/java/org/junit/platform/launcher/core/LauncherFactory.java

```diff
@@ -93,7 +94,13 @@ public static Launcher create(LauncherConfig config) throws PreconditionViolatio
 		}
 		engines.addAll(config.getAdditionalTestEngines());
 
-		Launcher launcher = new DefaultLauncher(engines);
+		Set<PostDiscoveryFilter> filters = new LinkedHashSet<>();
```

Comment:
❓ I'm not sure we should be using a `Set` here but then again it's consistent with `engines` above. Thoughts?

Reply:
Agree. Totally makes no sense to use set, only for consistency reasons.

By having a list instead of set we can kinda keep the order of filters. At some point having possibility to control the orders can give users some advantages

Reply:
replaced with list for now